packages/trpc/server/routers/viewer/organizations/update.handler.ts

```diff
@@ -15,6 +15,38 @@ import { TRPCError } from "@trpc/server";
 import type { TrpcSessionUser } from "../../../types";
 import type { TUpdateInputSchema } from "./update.schema";
 
+export const getBannerUrl = async (
+  banner: string | null | undefined,
+  teamId: number
+): Promise<string | null | undefined> => {
+  if (banner === undefined) {
+    // Banner not provided, don't update
+    return undefined;
+  }
+
+  if (banner === null) {
+    // Explicitly set to null, remove banner
+    return null;
+  }
```

Comment:
If we don't pass the bannerUrl but we return `undefined` in this function, won't this overwrite the stored value in the database?